README.md

```diff
@@ -12,6 +12,8 @@ Eclipse Theia Blueprint is a template for building desktop-based products based
 </div>
 
 [![Installers](https://img.shields.io/badge/download-installers-blue.svg?style=flat-curved)](https://theia-ide.org/docs/blueprint_download/)
+[![Build Status](https://ci.eclipse.org/theia/buildStatus/icon?subject=Main&job=Theia2%2Fmaster)](https://ci.eclipse.org/theia/job/Theia2/job/master/)
```

Comment:
Maybe "Latest" would be more intuitive than "Main"? Alternatively we could go with something more abstracted for both. We can also consider mentioning that we are referring to Theia with these. e.g.: 
`latest` <=> `next`
`stable` <=> `nightly`
`Theia (Stable)` <=> `Theia (Nightly)`

Reply:
I've updated the PR to `latest` and `next`: https://github.com/eclipse-theia/theia-blueprint/tree/jf/badges